en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -494,7 +494,10 @@ To create a new picklist:
 . Optionally, in the *Description* field, enter a description for the picklist. The following description is populated automatically based on your source and target systems: +
 `This maps [Source Field Name] from [Source Connection Name] to [Target Field Name] from [Target Connection Name]`
 . In the table, select the source fields and their target mappings.
-. Optionally, to add additional field mappings, click *Add Row* and enter the source field values manually (with a row representing each value) and select the target mappings from the dropdown.
+. Optionally, to add additional field mappings, click *Add Row* and if the additional field mapping is a:
+.. *Pick from List* field, enter the source field values manually (with a row representing each value) and select the target mappings from the dropdown.
+.. String field, manually define the value in the target field. 
```

Comment:
This one's correct for data type.

Reply:
Fixed the other ones and your comment disappeared - it is a field.